### PR TITLE
add com.ekonomikas.merkato

### PR DESCRIPTION
**Pull request by sheep-farm**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.ekonomikas.merkato.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+    "app-id" : "com.ekonomikas.merkato",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
```
Comment (hfiguiere) [inline com.ekonomikas.merkato.json:4]:
```suggestion
    "runtime-version" : "41",
```

3.38 is EOL

File: com.ekonomikas.merkato.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                }
+            ]
+        },
+        {
+            "name" : "com.ekonomikas.merkato",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "dir",
```
Comment (hfiguiere) [inline com.ekonomikas.merkato.json:51]: you need to download the source from somewhere

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by sheep-farm** [comment]
I accepted your suggestions. Thanks!

File: com.ekonomikas.merkato.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
+{
+    "app-id" : "com.ekonomikas.merkato",
```
Comment (hfiguiere) [inline com.ekonomikas.merkato.json:2]: the domain `ekonmikas.com` seems to be parked. Is it yours (or upstream) ?

**Comment by hfiguiere** [comment]
bot, build com.ekonomikas.merkato

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.ekonomikas.merkato.json (hunk 3 above)
Comment (sheep-farm) [reply com.ekonomikas.merkato.json:2]: The domain belongs to me.

Comment (hfiguiere) [reply com.ekonomikas.merkato.json:2]: ok

**Comment by hfiguiere** [comment]
```
Processing application com.ekonomikas.merkato
com.ekonomikas.merkato.desktop:    AppData problem: tag-missing : <description> required
Error loading AppData file: AppData file /app/share/appdata/com.ekonomikas.merkato.appdata.xml was not valid
```

**Comment by sheep-farm** [comment]
Resolved tag problem and appdata file completed.

**Comment by hfiguiere** [comment]
bot, build com.ekonomikas.merkato

**Comment by sheep-farm** [comment]
Well, no screenshots. 503 error.

File: com.ekonomikas.merkato.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            "name" : "com.ekonomikas.merkato",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dprofile=development"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/sheep-farm/merkato.git"
```
Comment (hfiguiere) [inline com.ekonomikas.merkato.json:54]: you need a commit here

**Comment by hfiguiere** [comment]
I think github is having issues at the moment hence the 503.

File: com.ekonomikas.merkato.json (hunk 4 above)
Comment (sheep-farm) [reply com.ekonomikas.merkato.json]: done.

**Comment by hfiguiere** [comment]
bot, build com.ekonomikas.merkato

**Comment by sheep-farm** [comment]
No more ERROR 503. Screenshot is avaliable.

**Comment by sheep-farm** [comment]
No more ERROR 503. Screenshot is avaliable.

**Comment by hfiguiere** [comment]
No. Right now it's 
```
• style-invalid         : Not enough <screenshot> tags, minimum is 1
```

**Comment by hfiguiere** [comment]
nevermind, let's try again (btw you can also do it)

bot, build com.ekonomikas.merkato

**Comment by sheep-farm** [comment]
"nevermind, let's try again (btw you can also do it)" Please tell me how.

**Comment by hfiguiere** [comment]
> "nevermind, let's try again (btw you can also do it)" Please tell me how.

bot, build com.ekonomikas.merkato

**Comment by sheep-farm** [comment]
test ok

**Comment by sheep-farm** [comment]
bot, build com.ekonomikas.merkato

**Comment by sheep-farm** [comment]
What happens now? How I can publish in Flathub?

File: com.ekonomikas.merkato.json [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy.git",
+                    "tag" : "1.4.0"
+                }
+            ]
+        },
+        {
+            "name" : "com.ekonomikas.merkato",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dprofile=development"
```
Comment (hfiguiere) [inline com.ekonomikas.merkato.json:49]: why developments? This should be release.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
An admin will review and eventually merge.

File: com.ekonomikas.merkato.json (hunk 5 above)
Comment (sheep-farm) [reply com.ekonomikas.merkato.json]: Removed. Thank you.

**Comment by sheep-farm** [comment]
bot, build com.ekonomikas.merkato

**Pull request closed without merging**